src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -100,6 +100,8 @@ public static string iOSBrokerKeySaveFailed(string keyChainResult)
 
         public const string UserRealmDiscoveryFailed = "User realm discovery failed. ";
 
+        public const string MsaNotSupportedWithRopc = "ROPC does not support MSA accounts. ";
```

Comment:
It's more the contrary? MSA accounts don't support ROPC ?
Or ROPC is not supported for MSA accounts? 
or ROPC is not supported on MSA accounts?

Reply:
yeah, i wasn't sure which. i was going for fewer words...but i get your point

Reply:
fixed. 

---
In reply to: [530722722](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2242#discussion_r530722722) [](ancestors = 530722722)